docs/supported-technologies.asciidoc

```diff
@@ -7,19 +7,74 @@ endif::[]
 
 == Supported technologies
 
-Support for the Elastic APM Node.js agent follows the support schedule of Node.js itself to the end-of-life period of each version after its maintenance term.
+The Elastic APM Node.js Agent automatically instruments various APIs in Node.js core and third-party frameworks and packages. This page lists all supported technologies and version ranges.
+
+
+[float]
+[[compatibility-node]]
+=== Node.js versions
+
+Support for the Elastic APM Node.js agent follows the https://nodejs.org/en/about/releases/[support schedule of Node.js itself]
+to the end-of-life period of each version after its maintenance term.
 Versions of Node.js past their end-of-life date are not supported.
 
 image::./images/node_release_schedule.svg[Node.js release schedule]
 
+Having said that, the current APM agent version (3.x) works with Node.js versions back to v8.6. We will only break support for older Node.js versions with a *major version release* of the APM agent.
```

Comment:
Do we need this?

```suggestion
The current APM agent version (3.x) works with Node.js versions back to v8.6. We will only break support for older Node.js versions with a *major version release* of the APM agent.
```

Reply:
I'm also not sure that the bold adds anything here. The purpose of the sentence is to explain when support breaks, but only bolding *major version release* doesn't emphasize that fact.